### DISCUSSION TITLE
Resizable: prints class names in Inspector

### DISCUSSION THREAD

**Discussion by fmaclen** [huntabyte/shadcn-svelte#998] [Help] [answered]
I'm posting this as a discussion because I don't know if this is an issue with `shadcn-svelte` or [`PaneForge`](https://github.com/svecosystem/paneforge).

In a brand-new project I added the `Resizable` component and every time the component loads it prints what appears to be class names to the logs:

```
cursor:ew-resize;touch-action:none;user-select:none;-webkit-user-select:none;-webkit-touch-callout:none;undefined
```

![image](https://github.com/huntabyte/shadcn-svelte/assets/1434675/683d1d98-90c7-4558-869f-29f0f7d0d5ab)

_Ignore the error in the screenshot, that's caused by an unrelated browser extension._

I added the component with this command:

```
npx shadcn-svelte@latest add resizable
```

And this is what the layout looks like this:

```svelte
<script lang="ts">
	import '../app.pcss';
	import * as Resizable from '$lib/components/ui/resizable/index.js';
</script>

<Resizable.PaneGroup direction="horizontal" class="h-screen w-screen">
	<Resizable.Pane minSize={15} maxSize={50} class="h-screen">sidenav</Resizable.Pane>
	<Resizable.Handle />
	<slot />
</Resizable.PaneGroup>
```

If a repo with the reproducible issue is needed let me know.

**Comment by fmaclen** [comment]
@huntabyte ping in case this case this flew under the radar.

**Reply by huntabyte** [reply, marked as answer]
Update to the latest version of paneforge

**Reply by fmaclen** [reply]
Thanks!
